Ryujinx.HLE/HOS/Services/ServerBase.cs

```diff
@@ -126,54 +126,60 @@ private void ServerLoop()
 
             while (true)
             {
-                int handleCount = _portHandles.Count + _sessionHandles.Count;
+                lock (_portHandles)
+                {
+                    lock(_sessionHandles)
```

Comment:
```suggestion
                lock (_portHandles)
                lock(_sessionHandles)
```
Would it also work if I write it like this? Just to reduce nesting

Reply:
To answer my own question: the IL looks the same, so I don't see any reason why it wouldn't work.